code/script.js

```diff
@@ -0,0 +1,107 @@
+//DOM-selectors 
+
+const imgWrapper = document.getElementById('imgWrapper')
+const userDataWrapper = document.getElementById('userDataWrapper')
+const projects = document.getElementById('projects')
+
+//Github API
+const username = 'Nosslexa'
+const API_URL = `https://api.github.com/users/${username}/repos`
+
+const options = {
+    method: 'GET',
+    headers: {
+          Authorization: 'API_TOKEN'
+      }
+  }
```

Comment:
I would suggest removing this as well since, I mentioned earlier that you don't seem to have a secret file and I think you can remove this code from line 11-16, since you don't seem to be using it in your script file.